frontend/packages/kubevirt-plugin/src/components/vm-disks/vm-disks.tsx

```diff
@@ -151,20 +152,24 @@ export const VMDisksTable: React.FC<React.ComponentProps<typeof Table> | VMDisks
   );
 };
 
-type VMDisksProps = {
-  vmLikeEntity?: VMGenericLikeEntityKind;
-  pvcs?: FirehoseResult<K8sResourceKind[]>;
-  datavolumes?: FirehoseResult<V1alpha1DataVolume[]>;
-  vmTemplate?: FirehoseResult<TemplateKind>;
+type VMDisksProps = VMTabProps & {
   vmi?: VMIKind;
 };
 
-export const VMDisks: React.FC<VMDisksProps> = ({ vmLikeEntity, vmTemplate, vmi }) => {
+export const VMDisks: React.FC<VMDisksProps> = ({ obj: vmLikeEntity, vmi }) => {
```

Comment:
please consider making this nested component in his own file

Reply:
IMO this is a bit out of scope of this PR. I suppose that we could split this file into more files, but that would get lost in the current folder which has 10 files already.

We probably need a better structure and larger refactoring to tackle other components in this folder as well.